documentation/continuous-integration.md

```diff
@@ -0,0 +1,34 @@
+<!-- @license CC0-1.0 -->
+
+# Continuous integration
+
+## Commit hashes for GitHub Actions
+
+We use commit hashes instead of version numbers for GitHub Actions in our workflows.
+
+Version numbers are based on Git tags, which are mutable.
+This means a maintainer can alter what code a tag like 'v1.0.0' refers to.
+If you use version number 'v1.0.0', you might download altered code without knowing it.
```

Comment:
Ah, I see. I only considered people creating a new release with bad code, and thought that wouldn’t happen with the first major release.

Still, using any other version number than 1.0.0 may indicate that this can happen at all of the version numbers?

Reply:
> ... and thought that wouldn’t happen with the first major release.

That might be a good reason to keep 'v1.0.0'? To indicate it can happen for all version numbers, even offical looking ones?

Or should we describe that more clearly?